### PR TITLE
feat: add support for BitBucket

### PR DESCRIPTION
**Pull request by ZeRego**
### Pre requirements:
- [x] I have followed (at least) the [PR section of the contributing guide](https://github.com/lightdash/lightdash/blob/main/.github/CONTRIBUTING.md#sending-a-pull-request).

### Reference:
Closes: #1004

### Description:
Add BitBucket support

### Preview:

Screenshot 2021-12-17 at 11 22 06

### Scope of the changes (select all that apply):
- [x] Frontend  
- [x] Backend  
- [ ] Common  
- [ ] E2E tests  
- [ ] Docs  
- [ ] CI/CD

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -243,7 +243,7 @@ When ready, access the development container and run these commands:
 
 ```shell
 # Connect to container
-docker exec -it lightdash-app-lightdash-dev-1 bash
+docker exec -it lightdash-app_lightdash-dev_1 bash
```
Comment (owlas) [inline .github/CONTRIBUTING.md:246]: Thanks!

File: packages/backend/package.json [hunk 2]
```diff
@@ -79,6 +79,7 @@
         "test": "jest",
         "lint": "eslint ./src",
         "format": "prettier --check --ignore-unknown ./src",
+        "create-migration": "knex migrate:make --knexfile src/database/knexfile.ts",
```
Comment (owlas) [inline packages/backend/package.json:82]: This is so useful, thank you!

**Review by owlas — APPROVED** [review]
Looks great! 

Feels like lot's of boilerplate having the separate project configs but I'm expecting these to diverge if we ever implement pull requests etc.

LGTM

**Comment by djayatillake** [comment]
🎉 my new co is on bitbucket so this is super helpful

**Pull request merged**